opentelemetry-appender-tracing/benches/logs.rs

```diff
@@ -34,8 +34,12 @@ struct NoopExporter {
 
 #[async_trait]
 impl LogExporter for NoopExporter {
-    async fn export(&self, _: LogBatch<'_>) -> LogResult<()> {
-        LogResult::Ok(())
+    #[allow(clippy::manual_async_fn)]
```

Comment:
This annotation is required for false positive - rust-lang/rust-clippy#12664

Reply:
Maybe have it on the project level until #12664 is addressed?

Reply:
Adding `#[allow(clippy::manual_async_fn)]` at the package level in `Cargo.toml` is not supported—it only works as an attribute in Rust source files. To suppress this lint effectively, it needs to be added in `lib.rs` or `mod.rs` for the main crate and separately for `benches` (and likely `tests` as well). It seems, adding at method level seems cleaner as of now. We can revisit this if similar implementation is done for Spans, and the issue is not fixed by then.